dags/chronos_train_dag.py

```
@@ -0,0 +1,344 @@
+    def load_data():
+        """
+        #### Load Data Task
+        Load data from CSV and convert to TimeSeriesDataFrame.
+        """
+        import pandas as pd
+        from autogluon.timeseries import TimeSeriesDataFrame
+        
+        logging.info("Starting load_data: Loading data from CSV")
+        try:
+            if not os.path.exists(DATA_PATH):
+                raise FileNotFoundError(f"Data file not found: {DATA_PATH}")
```

Ruff (0.12.2)

Abstract `raise` to an inner function

(TRY301)